server/services/mi-light/lib/light/light.configureBridge.js

```diff
@@ -0,0 +1,66 @@
+const logger = require('../../../../utils/logger');
+const { NotFoundError } = require('../../../../utils/coreErrors');
+const {
+  BRIDGE_EXTERNAL_ID_BASE,
+  BRIDGE_MODEL,
+  BRIDGE_IP_ADDRESS,
+  BRIDGE_MAC,
+  BRIDGE_NAME,
+  BRIDGE_TYPE,
+} = require('../utils/consts');
+
+/**
+ * @description Configure the milight bridge.
+ * @param {string} Mac - Mac Address of the Mi-Light Bridge.
+ * @example
+ * configureBridge('F0:FE:6B:A6:03:EB');
+ */
+async function configureBridge(Mac) {
+  const bridge = this.bridgesByMac.get(Mac);
+  if (!bridge) {
+    throw new NotFoundError(`BRIDGE_NOT_FOUND`);
+  }
```

Comment:
Is it really the expected error message ?

Reply:
I will be change with a clearer error message : No bridge discovered on the network with the following mac address : ${Mac}